Supersede "Improve Decidim Initiative documentation" (#6234)
decidim/decidim: decidim-initiatives/README.md

process and communication between the platform administrators and the sponsorship
committee is managed via an administration UI.

## Creating an initiative

> A user views the Initiatives page at `/initiatives`. If they are allowed to create an
> initiative and they belongs on a group, they see a call-to-action button: `New initiative`.
> 
> If there is more than one type of initiative, they will have to choose one and go through
> several screens, and go through a number of screens to help them refine their initiative.
> Once created, the initiative is not yet published.
> 
> The user must then edit their initiative and click the "Send for technical validation"
> button, which looks greyed out but can be clicked. Then it's sent to the site admins to
> review; once reviewed they can publish the initiative.

Reviewer: ```suggestion
A user views the Initiatives page at `/initiatives`. If she is allowed to create an
initiative and she belongs to a group, she will see a call-to-action button: `New initiative`.

If there is more than one type of initiative, she will have to choose one and go through
several screens to help her refine the initiative.
Once created, the initiative is not yet published.

The user must then edit her initiative and click the "Send for technical validation"
button, which looks greyed out but can be clicked. Then it's sent to the site admins to
review; once reviewed they can publish the initiative.
```

* online
* in person
* mixed

A PDF export of signatures is available.

#### Promotion committee

An initiative type can optionally be supported by a promotion committee, with a minimum
number of committee members. Once the user has created the initiative and before it can be
sent for technical validation they need to invite committee members to promote it.

When the user has created the initiative they will be given a link to share with possible
committee members, which will look something like `/initiatives/.../committee_requests/new`

When a prospective committee member opens the link, they can click a button which allows
them to request to be part of the committee. The initiative author then needs to approve
each request, by opening the Admin Dashboard link in the user menu, editing their
initiative, clicking "Committee members" and then approving each member.

Once enough people have joined the promoter committee the initiative author can send it for
technical validation.

## Usage

This plugin provides:

* A CRUD engine to manage initiatives.
* Public views for initiatives via a high level section in the main menu.

> * An admin dashboard for the initiative author
> * An admin dashboard for an initiative's promotion committee

Reviewer: Shouldn't we say
```suggestion
* An admin dashboard for the initiative's author and promotion committee
```
